UPCommons.js

```diff
@@ -96,8 +139,13 @@ var testCases = [
 						"creatorType": "author"
 					},
 					{
-						"firstName": "Miguel Angel",
-						"lastName": "Ahumada Sempoal",
+						"firstName": "Ahumada",
```

Comment:
I am not sure if we can , but it would make most sense to try to fix this in EM translator. Therefore I created this issue https://github.com/zotero/translators/issues/1333 to think further afterwards/besides. In the example here we have 9 DC tags each contains only one comma. This could also influence whether to split things up.